Sources/NIOHPACK/HuffmanCoding.swift

```diff
@@ -268,7 +268,7 @@ extension String {
 // this declaration on having the 5.3 compiler. This has caused a number of build issues. While updating
 // to newer Xcodes does work, we can save ourselves some hassle and just wait until 5.4 to get this
 // enhancement on Apple platforms.
-#if (compiler(>=5.3) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS))) || compiler(>=5.4)
+#if (compiler(>=5.3) && !(canImport(Darwin))) || compiler(>=5.4)
```

Comment:
We can drop this condition entirely now, as we no longer support Swift versions older than 5.6, so we will always enter this branch.

Reply:
Fixing this right now.